cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorage.kt

```diff
@@ -4,6 +4,9 @@ import android.content.SharedPreferences
 
 internal interface DataStorage : DataStorageGdpr, DataStorageCcpa {
     companion object {
+        const val KEY_LOCAL_DATA_VERSION = "sp.key.localDataVersion"
+        const val HARDCODED_LOCAL_DATA_VERSION = 1
```

Comment:
if you like the idea, you can call the 2 constants in a similar way so it is easy to look for them from the ide search i.e.
LOCAL_DATA_VERSION_KEY
LOCAL_DATA_VERSION_VALUE